### DISCUSSION TITLE
Como uso variables declaradas en bloques de codigo distintos con Jupyter? Es posible?

### DISCUSSION THREAD

**Discussion by RaulRU30** [LaunchX-InnovaccionVirtual/CursoIntroPython#74] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49573958/153346462-c3925062-6ef6-437e-81fa-4c486fc5e39a.png)

**Comment by RaulRU30** [comment]
Olvidenlo, ya vi que primero tenia que ejecutar el bloque de arriba para poder usar la variable xD

**Comment by JorgeEHE** [comment, marked as answer]
Hola Raul,
a lo que se puede ver en tu código en la imagen intentas dividir el texto en frases separándolas con '**.**', lo cual es una buena opción, pero no es necesario agregar las barras '**\\**'. Además te recomiendo agregar el texto dividido en una nueva variable para trabajar más fácil, por ejemplo:   

![image](https://user-images.githubusercontent.com/99061430/153349502-87912d01-c29a-46b6-96a0-dad1037b783a.png)


Por otro lado, y en base al titulo de tu publicación...
si entendí bien lo que estas preguntando te puedo decir esto:
las variables que se van declarando en los códigos de bloque se almacenan en la memoria y las puedes usar cuando las necesites, los valores asignado previamente se mantienen mientras no los alteres, espero este ejemplo te aclare la duda y sea entendible de mejor forma gráficamente como se muestra en las salidas  de los bloques 4 y 5 de la segunda imagen:

![image](https://user-images.githubusercontent.com/99061430/153349022-72217969-e8ca-4328-af75-696bf671525f.png)

![image](https://user-images.githubusercontent.com/99061430/153349083-c36072df-db6a-46b2-b7ac-80ea3150ee72.png)

**Reply by RaulRU30** [reply]
Si, justo despues de compartir mi problema me percate que no habia ejecutado el bloque de codigo donde se declaraba la variable, mil gracias <3
